blog/features/replica-signed-queries.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Announcing replica signed queries"
+description: To enhance the trustworthiness of queries by providing better guarantees against malicious attacks, we're excited to announce the release of replica signed queries!
```

Comment:
```suggestion
description: To enhance the trustworthiness of queries by providing better guarantees against attacks, we're excited to announce the release of replica signed queries!
```
The formulation “malicious attacks” seems not commonly used as attacks are malicious in nature. Maybe for simplicity, just attacks here.